### DISCUSSION TITLE
Stretch beyond chart limits

### DISCUSSION THREAD

**Discussion by jashan12345** [chrispahm/chartjs-plugin-dragdata#92] [General]
I Have Line Chart , Suppose there is data up to 100 amount and user want to drag beyond the 100 say 200 , how will it be possible for user to go beyond chart limits?

**Comment by chrispahm** [comment]
Hey @jashan12345,

unfortunately, I don't really see a simple way for achieving this. The most common way is to "force" axis boundaries using 

```js
const myChartOptions = {
  type: 'line', // or radar, bar, horizontalBar, bubble
  data: {...}, 
  options: {
    plugins: {dragData: true},
    scales: {
      y: {
        max: 25,
        min: 0
      }
    }
  }
}
```

This way, you can tell Chart.js to increase the padding of the chart to your desired range.
Is this helpful?

**Reply by jashan12345** [reply]
But this Padding Shows over the chart , suppose we have one record of value 10 and we want to drag over un imageable number  so would be it is possible ?

**Reply by chrispahm** [reply]
Unfortunately no, mostly because the internal method `getValueForPixel` does not return any (larger/smaller) values for outside the charts canvas. 
In the above fiddle you can see the axis scale re-adjusting once you drag the largest points to something smaller. Personally, I think this "jumpy" behaviour is bad UX, as it feels uncontrollable to a certain degree.

Maybe add a slider next to the chart controlling the axis scale?

**Reply by jashan12345** [reply]
ok so can you suggest me any better way to deal with this issue ?

**Reply by chrispahm** [reply]
Hmm maybe you can first describe your usecase a litte more?

**Reply by jashan12345** [reply]
ok any suggestion if value is unimageable

**Reply by chrispahm** [reply]
Still thinking this is bad UX though...
https://jsfiddle.net/6bw8e7fu/14/

**Reply by jashan12345** [reply]
ok Thanks For Helping me out , but I was finding To do this functionality with having slider ,

**Reply by chrispahm** [reply]
Not really, as I'm still not seeing any scenario where I would want users to drag to any value outside the charts boundaries.
Again, if you could explain such a scenario, there might be another more favorable approach to take

**Reply by jashan12345** [reply]
Suppose we want to forecast the data in that case user can drag outside the chart boundaries
